types/react-native/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for react-native 0.57
+// Type definitions for react-native 0.57.1
```

Comment:
This is why CI is red. DT only offers granularity to the MAJOR and MINOR version components, presumably because PATCH versions (according to semver) shouldn’t introduce new features.

Reply:
Ah. Thanks for the education!